tests/ImageSharp.Tests/Processing/Processors/Transforms/ResizeTests.cs

```diff
@@ -355,6 +355,7 @@ public void Resize_ThrowsForWrappedMemoryImage<TPixel>(TestImageProvider<TPixel>
         }
 
         [Theory]
+        [PlatformSpecific(~TestPlatforms.OSX)]
```

Comment:
@JimBobSquarePants reminder to ourselves that we shall open an issue for the disabled tests when this gets merged.

@AlexNDRmac I suppose you work on Mac. Can you help us by posting the output images saved by these tests on your machine? (`tests/Images/ActualOutput/ResizeTests/ResizeFromSourceRectangle_Rgba32_CalliphoraPartial.png`). Same for the other 2 disabled tests.

Expected images are under `tests/Images/External/ReferenceOutput/ResizeTests/` if interested.

Reply:
I propose to use save-action and fail scenario. For example: tests on mac failed, then starts build step with condition 'on-fail' and save any details (pictures, logs... etc) as build artifacts. Next, you can download it for investigation :)
I can add this scenario to workflow, it's very simple and useful.

Reply:
Sounds awesome, let's give it a try!
I wonder though what's simplest: here or in a separate PR?

Reply:
Do the tests not fail locally or something?

Reply:
I think, better - is separate PR, because this improvement not related to `libgdiplus`.

On my Mac some tests failed, but on CI such tests - passed :) I use 10.14.16, but CI using 10.15.x.... and on my Mac some tests kills some dotnet sub-processes... It's very interesting case, maybe I can foud whick component have "magic" influence on tests, but experiments requires some time.